cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -369,30 +371,53 @@ func ValidateHostPort(endpoint string, fldPath *field.Path) field.ErrorList {
 }
 
 // ValidateIPNetFromString validates network portion of ip address
-func ValidateIPNetFromString(subnet string, minAddrs int64, fldPath *field.Path) field.ErrorList {
+func ValidateIPNetFromString(subnetStr string, minAddrs int64, checkDualStack bool, fldPath *field.Path) field.ErrorList {
```

Comment:
can we please name the flag `isDualStackEnabled` (it's a common naming pattern).
[1]

Reply:
done.